libraries/config/src/config.cpp

```diff
@@ -144,6 +146,14 @@ FullNodeConfig::FullNodeConfig(Json::Value const &string_or_object, Json::Value
     network.network_boot_nodes.push_back(node);
   }
 
+  light_node_history = getConfigDataAsUInt(root, {"light_node_history"}, true, 0);
+
+  // Min history for light node is 110% of kDagExpiryPeriodLimit to avoid any race condition around the
+  // kDagExpiryPeriodLimit
+  if (light_node_history > 0 && light_node_history < kDagExpiryPeriodLimit * 1.1) {
```

Comment:
We have a void `FullNodeConfig::validate()` function in this file, which checks values limits, so this check should be probably there so we keep it on the same place + it should throw ConfigException rather than overriding the value so the node does not even start up and enforce user to setup this value properly in case he put there invalid number